src/components/AssocTable.vue

```diff
@@ -33,7 +33,7 @@
           slot="taxon"
           slot-scope="data"
         >
-          <i>{{ data.item.taxonLabel }}</i>
+          <a :href="'https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?id=' + data.item.taxonId"><i>{{ data.item.taxonLabel }}</i></a>
```

Comment:
Does this resolve? Assume this would make an incorrect url - for example for mouse - https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?id=NCBITaxon:10090

Reply:
good point, the one I didn't try. Do we just remove hyperlinks?